### ISSUE TITLE
crashes when a physical keyboard is connected 

### ISSUE THREAD

**Issue by hasanelfalakiy** [AndroidIDEOfficial/AndroidIDE#1782]
### Issue Checklist

- [X] I confirm that this issue is not a duplicate and hasn't been reported before.
- [X] I agree to follow the project's code of conduct.
- [X] I have checked and verified that I am using the latest version of AndroidIDE from GitHub or F-Droid.

### Additional terms

- [X] I understand that providing incomplete or inaccurate information may result in the issue being closed.
- [X] I agree to provide all requested details promptly for a comprehensive bug report.

### Issue explanation

the application crashes when a physical keyboard is connected, when connected it displays blank then I try to press tab but gboard reappears and gradle reloads 

### Expected behavior

I want when a physical keyboard is connected, the application display does not change (blank) & the application does not need to reload (gradle)

### Version name

Version : v2.7.1-beta (27102)
CI Build : true
Branch : main
Commit : 55bb46b31
Variant : arm64-v8a (release)
Build type : OFFICIAL
F-Droid Build : false
F-Droid Version : null
F-Droid Version code : -1
SDK Version : 30
Supported ABIs : [arm64-v8a, armeabi-v7a, armeabi]
Manufacturer : vivo
Device : vivo1901

### Download source

GitHub

### Relevant log output

```bash
AndroidIDE Crash Report
Version : v2.7.1-beta (27102)
CI Build : true
Branch : main
Commit : 55bb46b31
Variant : arm64-v8a (release)
Build type : OFFICIAL
F-Droid Build : false
F-Droid Version : null
F-Droid Version code : -1
SDK Version : 30
Supported ABIs : [arm64-v8a, armeabi-v7a, armeabi]
Manufacturer : vivo
Device : vivo1901

Stacktrace:
java.lang.IllegalStateException: Activity has been destroyed
	at com.itsaky.androidide.activities.editor.BaseEditorActivity.getBinding(Unknown Source:13)
	at androidx.appcompat.widget.Toolbar$3.prepareBuild(Unknown Source:126)
	at com.itsaky.androidide.utils.ApkInstaller$installApk$1.invoke(SourceFile:44)
	at com.itsaky.androidide.utils.ApkInstaller$installApk$1.invoke(SourceFile:129)
	at com.itsaky.androidide.tasks.TaskExecutorKt$$ExternalSyntheticLambda1.run(Unknown Source:19)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:257)
	at android.app.ActivityThread.main(ActivityThread.java:8192)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:626)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1015)
```

**Comment by BanDroid** [comment]
could you check if you disable soft keyboard when hard keyboard connected in your device settings?

```
System -> Language and Input -> Physical Keyboard
```

![Screenshot_20240401-050720.png](https://github.com/AndroidIDEOfficial/AndroidIDE/assets/68449470/52a325b4-0dd7-44ad-8160-56ebca3d4efe)

**Comment by hasanelfalakiy** [comment]
> could you check if you disable soft keyboard when hard keyboard connected in your device settings?
> 
> ```
> System -> Language and Input -> Physical Keyboard
> ```
> 
> ![Screenshot_20240401-050720.png](https://github.com/AndroidIDEOfficial/AndroidIDE/assets/68449470/52a325b4-0dd7-44ad-8160-56ebca3d4efe)
> 
> 

I've checked, my device doesn't have a physical keyboard setting like that

**Comment by itsaky** [comment]
This was fixed in 0801e90.
